homeassistant/components/denon/media_player.py

```diff
@@ -257,11 +257,12 @@ def volume_down(self):
 
     def set_volume_level(self, volume):
         """Set volume level, range 0..1."""
-        self.telnet_command("MV" + str(round(volume * self._volume_max)).zfill(2))
+        self.telnet_command(f"MV{str(round(volume * self._volume_max)).zfill(2)}")
```

Comment:
Zfilling the float can be handled in the f string directly.

https://stackoverflow.com/a/339013

Reply:
Fixed in #33667, precisely [here](https://github.com/home-assistant/core/pull/33667/files#diff-006b19d37922e226dac899bd7ea34b3a).